libs/librepcb/projecteditor/dialogs/projectsettingsdialog.ui

```diff
@@ -19,12 +19,24 @@
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
+     <property name="enabled">
+      <bool>true</bool>
+     </property>
+     <property name="layoutDirection">
+      <enum>Qt::LeftToRight</enum>
+     </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
+     <property name="tabsClosable">
+      <bool>false</bool>
+     </property>
+     <property name="tabBarAutoHide">
+      <bool>true</bool>
+     </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
-       <string>General</string>
+       <string/>
```

Comment:
There is still an empty tab, but now even with an empty name:
![project settings_010](https://user-images.githubusercontent.com/5374821/35415674-be1f3562-0226-11e8-9470-421c446fc25d.png)

So, just completely remove this tab, it's easy to add it again once we need it.